client/js/services/userService.js

```diff
@@ -37,6 +37,31 @@ ClonageApp.service('userService', ['socket', '$sessionStorage', function(socket,
         });
     }
 
+    function submitAnswer(enteredAnswer) {
+        emitAnswer(enteredAnswer);
+    }
+
+    function submitVote(enteredAnswer) {
+        alert(enteredAnswer);
+         emitVote(enteredAnswer);
+    }
+
+    function emitAnswer(answer){
+        socket.emit('USER answer', {
+            playerId: $sessionStorage.userId,
+            answer: answer,
+            roomId: $sessionStorage.roomId
```

Comment:
can just use the services's user variable to access the user id and room id, there are more likely to be up to date and can't be tampered with by Scott Logic peeps